src/adding-items-to-cart.spec.ts

```diff
@@ -36,4 +38,19 @@ describe('Add', () => {
         expect(await cart.numberOfItems).toEqual(1);
         expect(await cart.getItemTitle(0).getText()).toEqual(itemTitle);
     })
+
+    it('large numbers of items to cart', async () => {
+        await mainPage.changeCategory('Women');
+        await browser.wait(ExpectedConditions.visibilityOf(mainPage.searchCounter), 5000); //I wanted to find some unique element after the previous step, but the site is stupid))
+        await womenPage.pageItem.click();
+        await browser.wait(ExpectedConditions.visibilityOf(womenPage.quantityCounter), 5000);
+        await womenPage.quantityCounter.clear();
+        await womenPage.quantityCounter.sendKeys('#1000000000000000000');
+        await womenPage.addToCartButton.click();
+        await browser.sleep(10000);
```

Comment:
add browser.wait(ExpectedConditions...... instead of sleep

Reply:
fixed